dart/tool/service.md

```diff
@@ -1368,7 +1368,7 @@ class Instance extends Object {
   //
   // Provided for instance kinds:
   //   List
-  @Instance|Sentinel[] elements [optional];
+  @Instance[]|Sentinel[] elements [optional];
```

Comment:
Good catch - we should upstream and see what they say.

Note that we are using a slightly older version of service.md, so we should just patch this particular change in.